src/components/containers/newcharacter/NewCharacter.jsx

```diff
@@ -1,6 +1,7 @@
 import React, { Component } from 'react';
 import CharacterForm from '../../controls/CharacterForm';
 import  { postCharacter }  from '../../../services/character-api';
+import styles from './NewCharacter.css'
 
 
 export default class NewCharacter extends Component {
```

Comment:
while class components still work fine, I recommend getting used to functional components with hooks. Almost nobody in the industry uses class components